pkg/local_object_storage/metabase/containers.go

```diff
@@ -52,24 +53,25 @@ func (db *DB) containerSize(tx *bbolt.Tx, id *cid.ID) (uint64, error) {
 		return 0, err
 	}
 
-	key := id.ToV2().GetValue()
+	key := make([]byte, sha256.Size)
```

Comment:
By the way, what about adding `Size` constant to `oid` package? Using `sha256.Size` seems unobvious.

Reply:
Currently `id.Decode` docs require buffer to have at least 32 length. NeoFS API tells the same https://github.com/nspcc-dev/neofs-api/blob/f233a2fd6784b696b41741b4bcbb3bedaef2b066/refs/types.proto#L22.